.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,27 +1,60 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: ''
+title: "🐛 "
 labels: bug
-assignees: ''
-
+assignees: ""
 ---
 
 **Description**
+
 <!-- Write a clear and concise description of what the bug is. -->
 
 **Steps To Reproduce**
+
 <!-- Document the exact steps to reproduce the behavior in a numbered list, like so:
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error -->
 
 **Expected behavior**
+
 <!-- Write a clear and concise description of what you expected to happen. -->
 
 **Screenshots**
+
 <!-- If applicable, add screenshots to help explain your problem. -->
 
 **Logs**
- <!-- Post the full error log here -->
+
+<details>
+  <summary>Logs</summary>
+
+<!--
+      Run your application with `flutter run --verbose` and attach all the
+      log output below between the lines with the backticks. If there is an
```

Comment:
We should simply say "Post all relevant logs related to your bugs" or similar, rather than "If there is an exception..."

Reply:
But the whole point of posting the logs of `flutter run --verbose` is to post the whole logs. Thats why it's wrapped in a `<details>` tag